modules/ref-inference-endpoints.adoc

```diff
@@ -87,14 +87,20 @@ vLLM ServingRuntime for KServe::
 * `:443/detokenize`
 +
 [NOTE]
---
-The vLLM runtime is compatible with the OpenAI REST API. For a list of models that the vLLM runtime supports, see link:https://docs.vllm.ai/en/latest/models/supported_models.html[Supported models].
---
+====
+* The vLLM runtime is compatible with the OpenAI REST API. For a list of models that the vLLM runtime supports, see link:https://docs.vllm.ai/en/latest/models/supported_models.html[Supported models].
+* To use the embeddings inference endpoint in vLLM, you must use an embeddings model that the vLLM supports. You cannot use the embeddings endpoint with generative models. For more information, see link:https://github.com/vllm-project/vllm/pull/3734[Supported embeddings models in vLLM].
+* As of vLLM v0.5.5, you must provide a chat template while querying a model using the `/v1/chat/completions` endpoint. If your model does not include a predefined chat template, you can use the `chat-template` command-line parameter to specify a chat template in your custom vLLM runtime, as shown in the example. Replace `<CHAT_TEMPLATE>` with the path to your template.
 +
-[NOTE]
---
-To use the embeddings inference endpoint in vLLM, you must use an embeddings model that the vLLM supports. You cannot use the embeddings endpoint with generative models. For more information, see link:https://github.com/vllm-project/vllm/pull/3734[Supported embeddings models in vLLM].
---
+[source]
+----
+containers:
+  - args:
+      - --chat-template=<CHAT_TEMPLATE>
+----
++
+You can also use the chat templates included with the vLLM image under `/apps/data/template`.
```

Comment:
Templates are available as `.jinja` files in https://github.com/opendatahub-io/vllm/tree/main/examples, I'd add a link so that users can get a list of the available templates.

We should probably also add some more context regarding chat templates, for example we could add the huggingface docs page: https://huggingface.co/docs/transformers/main/chat_templating

Reply:
@dtrifiro Updated in latest commit